assets/js/components/Dashboard/NewPaMessage/StationsAndZones/SelectZonesPage.tsx

```diff
@@ -279,25 +274,23 @@ const SelectZonesPage = ({
                   className={cx("button-secondary-outline", {
                     "button-active": isAllSelected,
                   })}
-                  onClick={() =>
-                    selectGroups([LEFT_ZONES, MIDDLE_ZONES, RIGHT_ZONES])
-                  }
+                  onClick={() => selectDirectionGroup([0, 1])}
```

Comment:
Would this be equivalent to "select everything"? (i.o.w. is there even a need for `selectDirectionGroup` to support more than one direction?)

Reply:
Good point. This case doesn't need this function.